src/TSID/include/BipedalLocomotion/TSID/TaskSpaceInverseDynamics.h

```diff
@@ -0,0 +1,114 @@
+/**
+ * @file TaskSpaceInverseDynamics.h
+ * @authors Ines Sorrentino
+ * @copyright 2021 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_TASK_SPACE_INVERSE_DYNAMICS_H
+#define BIPEDAL_LOCOMOTION_TASK_SPACE_INVERSE_DYNAMICS_H
+
```

Comment:
Remember to include what you use

Reply:
https://include-what-you-use.org/

Reply:
```suggestion
#include <optional>
#include <functional>
```


functional for `reference_wrapper`